grails-app/assets/stylesheets/homepage.css

```
@@ -0,0 +1,134 @@
+body {
+    background-color: #0030B9;
+    margin: 0px;
+}
+
+html, body {
+    height: 100%;
+}
+
+.page-container {
+    display: flex;
+    align-items: center;
+}
+
+.left-bar {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+    width: 33%;
+}
+
+.asaas-logo {
+    width: 10.9375rem;
+    height: auto;
+    margin-left: 1.25rem;
+    margin-top: 0.9375rem;
+}
+
+.cellphone {
+    width: 25rem;
+    height: auto;
+    margin-left: 1.875rem;
+}
+
+.center {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+    width: 33%;
+}
+
+.top-text {
+    margin-top: 8.125rem;
+    margin-bottom: 8.125rem;
+    color: #FFFFFF;
+    font-family: sans-serif;
+    font-weight: bold;
+    text-align: center;
+    letter-spacing: .004rem;
+}
+
+.card-and-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    height: 100%;
+    width: 100%;
+}
+
+.card-buttons {
+    width: 31.25rem;
+    height: 12.5rem;
+    background-color: white;
```

```suggestion
    background-color: #FFFFFF;
```
Pra seguir o mesmo padrão que as outras declarações